### ISSUE TITLE
Conflicting re-exports error

### ISSUE THREAD

**Issue by rsheasby** [roxiness/routify#307]
**bug description**
When building using 2.7.4, I get the following compile warning:

```
(!) Conflicting re-exports
node_modules/@roxi/routify/runtime/index.js re-exports 'isChangingPage' from both node_modules/@roxi/routify/runtime/store.js and node_modules/@roxi/routify/runtime/helpers.js (will be ignored)
```
Everything still seems to work, but it's just a bit annoying to have the error on every compile.

**package.json**
Running the following command: `routify -b && rollup -c`

**version**
Error occurs with any version later than 2.7.4 it seems. I've tried 2.7.5, 2.7.6, and 2.8.3

**Comment by ghostdevv** [comment]
We know of this issue and it is going to get fixed in a further path, thank you for making a official issue about it

**Comment by jakobrosenberg** [comment]
This should be fixed in `2.8.4`.

**Comment by ghostdevv** [comment]
If you have this issue again feel free to reopen
